process_preds.Rmd

```diff
@@ -0,0 +1,116 @@
+---
+title: "Code to process predictors for urban niche model"
+author: "Mikko Jimenez, Josh Carrell, and Caitlin Mothes"
+date: "'r Sys.Date()'"
+output: html_notebook
+---
+
+## Process predictors
+
+As part of a broader pipeline for running a niche model in urban settings, this script is designed to provide a flexible process for reading in predictor variables, setting them to a consistent coordinate reference system, and cropping them to an extent of interest. 
+
+Note: it is currently unclear if predictors will be expected to be downloaded prior to running this script or if we will be pulling predictors directly from an API. I've written code for each situation below. 
+
+## set up environment
+```{r}
+source("setup.R")
+
+# read in spatial layer that includes full extent of interest, in this case a boundary .shp
+# having issues with symlink, run by Caitlin
```

Comment:
Happy to help if there are still symlink issues!

Reply:
We sorted this out together already, just forgot to remove this annotation!